modules/force_rtl.py

```diff
@@ -0,0 +1,52 @@
+"""
+Forces drone to return to launch (RTL).
+"""
+
+import dronekit
+
+from . import upload_commands
+
+MAVLINK_RTL_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL
+MAVLINK_RTL_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_RETURN_TO_LAUNCH
+
+
+def force_rtl(drone: dronekit.Vehicle) -> bool:
+    """
+    Sends RTL command using the upload_command module.
+    
+    Parameters
+    -----------
+    drone: dronekit.Vehicle
+        The connected drone.
+
+    Returns
+    -------
+    bool: True if uploading RTL command is successful, False otherwise.
+    """
+
+    # clears current mission
+    rtl_command = dronekit.Command(
```

Comment:
If you update your branch we have a `generate_command` module. You can use that to create the RTL command.
```
rtl_command = generate_command.return_to_launch()
```